lib/stormpath-sdk/provider/github/github_provider.rb

```diff
@@ -14,5 +14,5 @@
 # limitations under the License.
 #
 class Stormpath::Provider::GithubProvider < Stormpath::Provider::Provider
-  prop_reader :client_id, :client_secret
+  prop_reader :client_id, :client_secret, :user_info_mapping_rules
```

Comment:
It says in the specs '`The directory's provider endpoint, should have a reference to the attributeMapping (just like in case of SAML). `'

and in the `SamlProvider` the mapping rules are defined as a property reader, not an association:

```
prop_reader :attribute_statement_mapping_rules
```